curriculum/challenges/english/05-apis-and-microservices/mongodb-and-mongoose/create-many-records-with-model.create.english.md

```diff
@@ -12,7 +12,8 @@ Sometimes you need to create many instances of your models, e.g. when seeding a
 
 ## Instructions
 <section id='instructions'>
-Create many people with <code>Model.create()</code>, using the function argument <code>arrayOfPeople</code>.
+Add a function expecting an <code>arrayOfPeople</code> argument which creates many people through <code>Model.create()</code>.
```

Comment:
```suggestion
Modify the <code>createManyPeople</code> function to create many people through <code>Model.create()</code> and the argument <code>arrayOfPeople</code>.
```
The function already exists. If a user creates a function with a different name, they would need to modify code in sections we've told them not to touch.